### ISSUE TITLE
Model for SDXL?

### ISSUE THREAD

**Issue by foureyednymph** [open-mmlab/PIA#10]

**Comment by kowalgregy** [comment]
+1

**Comment by johndpope** [comment]
<img width="1681" alt="Screenshot 2024-01-07 at 7 23 22 am" src="https://github.com/open-mmlab/PIA/assets/289994/43f94610-9b7a-401d-bcc6-aeb67072c0a8">

upstream animatediff has sdxl support 
I guess the pia checkpoint would need to be retrained for higher res?

**Comment by zengyh1900** [comment]
Yes. If we would like to use PIA on SDXL, it requires re-training on SDXL. 
Unfortunately, we don't have any plans for the moment.  
Volunteers are always welcome! :cupid: :cupid:
